CHANGELOG.md

```diff
@@ -26,6 +26,8 @@ Read our [guidelines for writing a good changelog entry](https://github.com/biom
 
 - Fix [#1173](https://github.com/biomejs/biome/issues/1173). Fix the formatting of a single instruction with commented in a control flow body to ensure consistency. Contributed by @mdm317
 
+- Fix overriding of `javascript.globals`. Contributed by @arendjr
```

Comment:
This could be nice of clarifying what is fixed.

Reply:
I find it difficult to clarify this further 😅 Using `javascript.globals` in an override pattern didn’t work. As in, it was ignored completely.